### PR TITLE
DC_226 prisoner background page

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/recat/PrisonerBackgroundSpecification.groovy [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  def setup() {
+    db.clearDb()
+  }
+
+  TestFixture fixture = new TestFixture(browser, elite2Api, oauthApi, riskProfilerApi)
+  DatabaseUtils db = new DatabaseUtils()
+
+  def "The page displays all warnings and saves details correctly"() {
+    when: 'I go to the Higher Security Review page'
```
Comment (steverendell) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/recat/PrisonerBackgroundSpecification.groovy]: copied when text !

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/recat/PrisonerBackgroundSpecification.groovy [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    def data = db.getData(12)
+    def response = new JsonSlurper().parseText(data.form_response[0].toString())
+    data.status == ['STARTED']
+    data.cat_type.value == ['RECAT']
+    response.recat == [prisonerBackground: [offenceDetails: 'offenceDetails text']]
+    data.user_id == ['RECATEGORISER_USER']
+    data.assigned_user_id == ['RECATEGORISER_USER']
+  }
+
+  def ''() {
```
Comment (steverendell) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/recat/PrisonerBackgroundSpecification.groovy]: blank method name!

File: server/services/offendersService.js [hunk 3]
```diff
@@ -455,6 +455,38 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
...
 
+  async function getPrisonerBackground(token, offenderNo) {
+    try {
+      const nomisClient = nomisClientBuilder(token)
+      const currentCats = await nomisClient.getCategoryHistory(offenderNo)
+
+      const decoratedCats = await Promise.all(
+        currentCats.map(async o => {
+          const description = await getOptionalAssessmentAgencyDescription(nomisClient, o.assessmentAgencyId)
+          const assessmentMoment = moment(o.assessmentDate, 'YYYY-MM-DD')
```
Comment (steverendell) [inline server/services/offendersService.js]: dateConverter() will sort this , its available in the html too

File: server/views/formPages/recat/prisonerBackground.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+  {% if data.extremismProfile.notifyRegionalCTLead %}
+  <p class="govuk-heading-s">Please notify your regional CT lead that you have this person in custody</p>
+  {% endif %}
+
+<h1 class="govuk-heading-m">Other sources to check</h1>
+  <p>
+    If available, you may want to manually check the following sources:
+  <ul>
+    <li>OAYsis</li>
```
Comment (steverendell) [inline server/views/formPages/recat/prisonerBackground.html]: think its Oasys

File: server/services/offendersService.js (hunk 3 above)
Comment (lbennett-moj) [reply server/services/offendersService.js]: there are a few dotted around - I'll raise a minor tech debt ticket for this

**Pull request merged**